src/main/resources/application.yml

```diff
@@ -116,4 +116,10 @@ hmpps:
     queueAdminRole: COURT_CASE_MATCHER_QUEUE_ADMIN
 
 replay404:
-  dry-run: true
\ No newline at end of file
+  dry-run: true
+
+commonplatform:
```

Comment:
This does not seem like the kind of thing which is likely to change per environment or indeed ever - could potentially be hard-coded